apps/docs/stories/components/CheckoutWrapper/Docs.mdx

```diff
@@ -0,0 +1,105 @@
+import {
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  ArgTypes,
+  Source,
+} from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExample } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiCheckoutWrapper from './index.stories.tsx';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Checkout Wrapper"
+  Components="justifi-checkout-wrapper"
+  of={JustifiCheckoutWrapper}
+/>
+
+<Title />
+
+---
+
+Component to be used as a container for the checkout components. It handles tokenization of payment methods, billing information, insurance and submission of the checkout form.
+It also provides a way to save the payment method for future use.
+
+Components that can be used inside the wrapper include:
```

Comment:
I'd change this to be more explicit:
`These sub components can be added to the checkout wrapper:`